packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1572,6 +1572,28 @@ Scenario that returns a different file encoding depending on the accept header.
 - image/png return a png image
 - image/jpeg return a jpeg image
 
+### Payload_MultiPart_FormData_basic
+
+- Endpoint: `post /multipart/form-data/mixed-parts`
+
+Expect request:
+
+```
+POST /upload HTTP/1.1
+Content-Length: 428
+Content-Type: multipart/form-data; boundary=abcde12345
+
+--abcde12345
+Content-Disposition: form-data; name="id"
+
+123
+--abcde12345
+Content-Disposition: form-data; name="profileImage ";
```

Comment:
```suggestion
Content-Disposition: form-data; name="profileImage ";
Content-Type: application/octet-stream;
```

Reply:
When there is no content-type defined for `profileImage` in tsp, we will not add the line `Content-Type: application/octet-stream;` in the part.

Reply:
Yeah. Typespec doesn't support to define content-type for each item so I don't check content-type in the test case. We may add it in the future.

Reply:
Update:
According to https://datatracker.ietf.org/doc/html/rfc7578#section-4.4, we shall add content-type in test case